java/src/main/native/src/row_conversion.cu

```diff
@@ -923,106 +910,89 @@ __global__ void copy_from_rows(const size_type num_rows, const size_type num_col
   // to speed up some of the random access memory we do, we copy col_sizes and col_offsets
   // to shared memory for each of the tiles that we work on
 
-  constexpr unsigned stages_count = NUM_TILES_PER_KERNEL_LOADED;
   auto group = cooperative_groups::this_thread_block();
-  extern __shared__ int8_t shared_data[];
-  int8_t *shared[stages_count] = {shared_data, shared_data + shmem_used_per_tile};
+  auto warp = cooperative_groups::tiled_partition<cudf::detail::warp_size>(group);
```

Comment:
Can this be `constexpr`?

Reply:
They can't be constexpr, but the can be const. Changed that.